addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.xml

```diff
@@ -105,92 +65,69 @@
                 <div class="flex-grow-1"/>
                 <t t-call="mail.DiscussSidebarChannel.commands"/>
                 <t t-foreach="indicators" t-as="indicator" t-key="indicator_index" t-component="indicator" t-props="{ thread }"/>
                 <i t-if="thread.isMuted" class="fa fa-fw fa-bell-slash me-1"/>
-            </t>
-            <t t-else="">
-                <div t-if="indicators.length" class="position-absolute rounded-circle p-0 smaller o-mail-DiscussSidebarChannel-indicatorCompact lh-1 bg-inherit" name="indicator-compact">
-                    <t t-component="indicators[0]" t-props="{ thread }"/>
-                </div>
-            </t>
             <t t-if="thread.selfMember?.message_unread_counter > 0 and !thread.isMuted and thread.importantCounter === 0" t-call="mail.DiscussSidebar.unreadIndicator"/>
-            <span t-if="thread.importantCounter > 0" t-attf-class="o-mail-DiscussSidebarChannel-badge o-mail-DiscussSidebar-badge badge rounded-pill o-discuss-badge flex-shrink-0 fw-bold {{thread.isMuted ? 'o-muted' : ''}}" t-att-class="{ 'mx-1': !store.discuss.isSidebarCompact, 'position-absolute top-0 o-compact': store.discuss.isSidebarCompact }" t-esc="thread.importantCounter"/>
+            <span class="mx-1" t-if="thread.importantCounter > 0" t-attf-class="o-mail-DiscussSidebarChannel-badge o-mail-DiscussSidebar-badge badge rounded-pill o-discuss-badge flex-shrink-0 fw-bold {{thread.isMuted ? 'o-muted' : ''}}" t-esc="thread.importantCounter"/>
         </button>
     </t>
 
     <t t-name="mail.DiscussSidebarChannel.commands">
-        <div class="o-mail-DiscussSidebarChannel-commands" t-att-class="{
-            'd-none': !store.discuss.isSidebarCompact,
-            'mx-1': !store.discuss.isSidebarCompact and thread.importantCounter === 0,
-            'ms-1': !store.discuss.isSidebarCompact and thread.importantCounter > 0,
+        <div class="o-mail-DiscussSidebarChannel-commands d-none" t-att-class="{
+            'mx-1': thread.importantCounter === 0,
+            'ms-1': thread.importantCounter > 0,
         }">
-            <div t-att-class="{ 'd-flex flex-column align-items-start ps-1 pt-1': store.discuss.isSidebarCompact, 'btn-group btn-group-sm o-gap-0_5': !store.discuss.isSidebarCompact }">
+            <div class="btn-group btn-group-sm o-gap-0_5">
                 <t t-foreach="sortedCommands" t-as="command" t-key="command_index">
-                    <button class="btn o-opacity-35 opacity-100-hover" t-att-class="{ 'd-flex align-items-center px-1 py-0 gap-1 text-start smaller btn-secondary': store.discuss.isSidebarCompact, 'btn-light rounded-3 p-0 border-transparent': !store.discuss.isSidebarCompact }" t-on-click.stop="() => command.onSelect()" t-att-title="store.discuss.isSidebarCompact ? '' : command.label">
+                    <button class="btn opacity-50 opacity-100-hover btn-light rounded-3 p-0" t-on-click.stop="() => command.onSelect()" t-att-title="command.label">
```

Comment:
was `o-opacity-35`, why change to `opacity-50`? This is an unrelated change to the PR at hand. Again this was made intentionally.